src/github/mod.rs

```diff
@@ -1,9 +1,61 @@
 pub(crate) mod graphql;
 
 use color_eyre::eyre::{eyre, WrapErr};
+use serde::{Deserialize, Serialize};
 
 use crate::build_http_client;
 
+const GITHUB_ACTOR_TYPE_USER: &str = "User";
+const GITHUB_ACTOR_TYPE_ORGANIZATION: &str = "Organization";
+
+#[derive(Serialize, Deserialize)]
+pub struct WorkflowData {
+    event: WorkflowDataEvent,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct WorkflowDataEvent {
+    repository: WorkflowDataEventRepo,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct WorkflowDataEventRepo {
+    owner: WorkflowDataEventRepoOwner,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct WorkflowDataEventRepoOwner {
+    login: String,
+    #[serde(rename = "type")]
+    kind: String,
+}
+
+pub(crate) fn get_actions_event_data() -> color_eyre::Result<WorkflowData> {
+    let github_context = std::env::var("GITHUB_CONTEXT")?;
```

Comment:
Probably wrap_err here to say "GITHUB_CONTEXT was not found in environment" or something?

Reply:
:shrug: maybe, I feel like it's good enough, it prints a generic message anyway.

Reply:
Will it be good enough when we have to debug it? :P